src/pluginHooks.js

```diff
@@ -1460,9 +1469,66 @@ const REGISTERED_HOOKS = [
    * ```
    */
   'afterFilter',
-
   /* eslint-enable jsdoc/require-description-complete-sentence */
 
+  /**
+   * Called when a value is updated in the engine.
+   *
+   * @since 9.0.0
+   * @event Hooks#afterFormulasValuesUpdate
+   * @param {Array} changes The values and location of applied changes.
+   */
+  'afterFormulasValuesUpdate',
+
+  /**
+   * Called when a named expression is added to the Formulas' engine instance.
+   *
+   * @since 9.0.0
+   * @event Hooks#afterNamedExpressionAdded
+   * @param {string} namedExpressionName The name of the added expression.
```

Comment:
It seems that the hook signature is changed https://github.com/handsontable/handsontable/blob/feature/hyperformula-2021/src/plugins/formulas/formulas.js#L527. The `namedExpressionName` argument is not passed anymore.

Reply:
Ahh then d.ts file needs to be updated https://github.com/handsontable/handsontable/blob/feature/hyperformula-2021/handsontable.d.ts#L1802.

Reply:
It is passed in https://github.com/handsontable/handsontable/blob/f1dd40a2b008f765736454ded538bcbd01149dfb/src/plugins/formulas/formulas.js#L539

Reply:
You are actually right 😄  I had to look at another hook.